src/contracts/BCoWFactory.sol

```diff
@@ -25,11 +25,17 @@ contract BCoWFactory is BFactory {
    * @dev Deploys a BCoWPool instead of a regular BPool, maintains the interface
    * to minimize required changes to existing tooling
    */
-  function newBPool() external override returns (IBPool _pool) {
+  function newBPool() external override(BFactory, IBFactory) returns (IBPool _pool) {
     IBPool bpool = new BCoWPool(SOLUTION_SETTLER, APP_DATA);
     _isBPool[address(bpool)] = true;
     emit LOG_NEW_POOL(msg.sender, address(bpool));
     bpool.setController(msg.sender);
     return bpool;
   }
+
+  /// @inheritdoc IBCoWFactory
+  function emitEvent() external {
```

Comment:
`logBPool()` ?
thinking of how not to be so direct with this naming: `emitEvent`